README.md

```diff
@@ -8,26 +8,33 @@ Install with a single command
 sh -c "$(curl -fsSL https://raw.githubusercontent.com/cyral-quickstart/quickstart-sidecar-poc/main/install-poc.sh)"
 ```
 
-# Deploy an Instance
+# Cloud Deployment Instructions
+
+The above command will work on just about any system, but you can follow the below directions to create a new instance for the sidecar.
 
 ## AWS EC2
 
 1. Go to [EC2 Service](https://console.aws.amazon.com/ec2)
 1. Select [Launch Instance](https://console.aws.amazon.com/ec2/v2/home#LaunchInstances) and provide the following info
     1. Name: Provide something meaningful like CyralSidecar
-    1. Amazon Machine Image (AMI): The default Amazon Linux image and options are fine, but most images should work fine
+    1. Amazon Machine Image (AMI): The default Amazon Linux image and options are fine, but most images should work
     1. Instance Type: Our recommended flavor is M5.large, but a T3 or T2 large will work well for a POC as well
     1. Key Pair: Select or create one
-    1. Security Group: You can use a precreated one or create a new one. You'll want to make sure it has SSH access. *Make note of the name*, we'll have to modify it for the DB Ports
+    1. Network Settings: Utilize the Edit button on the section header to create a new Security Group for this POC
+        1. Make sure Create Security Group is selected
+        1. Security Group Name: Provide a useful name like cyral-sidecar-poc
+        1. Description: This is required so provide a description
+        1. Inbound Security Rules:
+            1. ssh - This rule should already exist, but review the Source Type and Source to make sure its appropriate for your environment
+            1. Add Security Group Rule: One per DB type you'd like to test 
+                1. Type: Custom TCP
+                1. Port Range: Enter the port number you'd like to use, for example if you're testing PostgreSQL you might want to use 5432
+                1. Source Type / Source: Provide approrpriate values that will allow your DB Client to connect to this port.
```

Comment:
For "Source" can they put a subnet range that covers all their clients? Maybe just writing "clients" plural will be enough to convey this.